openedx_events/learning/signals.py

```diff
@@ -7,3 +7,84 @@
 They also must comply with the payload definition specified in
 docs/decisions/0003-events-payload.rst
 """
+
+from openedx_events.learning.data import (
+    CertificateData,
+    CohortData,
+    CourseEnrollmentData,
+    RegistrationFormData,
+    StudentData,
+)
+from openedx_events.tooling import OpenEdxPublicSignal
+
+STUDENT_REGISTRATION_COMPLETED = OpenEdxPublicSignal(
```

Comment:
Have you considered how these will be documented? Inline annotation documentation has the benefit of being close to the code, and would allow documentation to be added immediately along with the event definitions. 

See toggles documentation as an example: https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html

Thanks.

Reply:
Excellent suggestion! We'll give it a go!

I'll tag you when it's ready!